src/panels/lovelace/cards/energy/hui-energy-sources-table-card.ts

```diff
@@ -1109,6 +1138,12 @@ export class HuiEnergySourcesTableCard
     </ha-card>`;
   }
 
+  private _handleMoreInfo(ev): void {
+    fireEvent(this, "hass-more-info", {
+      entityId: ev.currentTarget?.entity,
+    });
```

Comment:
We probably want to skip this if it is an external statistic (w/ no entity id). See `isExternalStatistic()`